### PR TITLE
test: add unset pids limit test

### PR DESCRIPTION
**Pull request by haircommander**
#### Does this PR introduce a user-facing change?

/kind ci
```release-note
none
```

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
@haircommander Is [this](https://github.com/cri-o/cri-o/pull/4614) PR related to the change?

**Comment by haircommander** [comment]
hah I forgot about that one. I will rebase that and we can close this

**Comment by haircommander** [comment]
per @sohankunkerkar 's suggestion, we're carrying https://github.com/cri-o/cri-o/pull/4614 here

File: test/cgroups.bats [hunk 1]
```diff
@@ -29,19 +29,40 @@ monitor_cgroup = "$MONITOR_CGROUP"
...
+	fi
+	run_pids_limit_test "-1" "max"
+}
+
+@test "zero pids limit" {
+	if ! grep -qEw ^pids /proc/cgroups; then
+		skip "pids cgroup controller is not available"
+	fi
+
+	run_pids_limit_test "0" "max"
```
Comment (haircommander) [inline test/cgroups.bats:65]: this will fail in runc without https://github.com/opencontainers/runc/pull/4015

**Pull request closed without merging**